app/layouts/main_layout.py

```diff
@@ -62,15 +68,45 @@ def get_main_layout():
                     }
                 ),
             ),
-            dcc.Store(id="event_id_on_display", data=0),
+            dcc.Store(
+                id="media_url",
+                storage_type="session",
+                data={},
+            ),
+            dcc.Store(id="wildfire_id_on_display", data=0),
             dcc.Store(id="auto-move-state", data={"active": True}),
             # Add this to your app.layout
             dcc.Store(id="bbox_visibility", data={"visible": True}),
+            dbc.Modal(
```

Comment:
not related with this pr

Reply:
hum I don't think I have added this, maybe when I  tried to sync with main. Should I removed this or it's currently in main ?